nhais-adaptor/edifact/incoming/parser/deserialiser.py

```diff
@@ -0,0 +1,100 @@
+import edifact.incoming.parser.creators as creators
+from edifact.incoming.models.message import MessageSegment
+from edifact.incoming.models.interchange import Interchange
+
+INTERCHANGE_HEADER_KEY = "UNB"
+MESSAGE_HEADER_KEY = "UNH"
+MESSAGE_BEGINNING_KEY = "BGM"
+MESSAGE_REGISTRATION_KEY = "S01"
+MESSAGE_PATIENT_KEY = "S02"
+MESSAGE_TRAILER_KEY = "UNT"
+INTERCHANGE_TRAILER_KEY = "UNZ"
+
+
+def extract_relevant_lines(original_dict, starting_pos, trigger_key):
+    """
+    From the original dict generate a smaller dict just containing the relevant lines based upon the trigger key
+    will keep looping till the terminating key is found in the terminating config

```

Comment:
You might need a full stop between this line and the previous one.

Reply:
👍 I'll keep an eye out. I imagine this isn't the only place where I've left a full stop out